book/chapters/appendices/solutions_large-scale_benchmarking.qmd

```diff
@@ -155,40 +150,35 @@ submitJobs(reg = reg)
 waitForJobs(reg = reg)
 ```
 
-After the execution of the experiment finished, we can collect the results like below.
+After the execution of the experiment finished we can load the results as a `r ref("BenchmarkResult")`.
 
 ```{r solutions-040}
 bmr = reduceResultsBatchmark(reg = reg)
-bmr
+bmr$aggregate(msr("regr.mse"))
 ```
 
-6. Conduct a global Friedman test and interpret the results using `regr.mse`. Why do we not need to use the post-hoc test?
+3. Conduct a global Friedman test and, if appropriate, post hoc Friedman-Nemenyi tests, and interpret the results.
+  As an evaluation measure, use the MSE.
 
-As a first step, we load `r ref_pkg("mlr3benchmark")` and create a benchmark aggregate using `msr("regr.mse")`.
+First, we load the `r ref_pkg("mlr3benchmark")` package and create a `r ref("mlr3benchmark::BenchmarkAggr")` from the benchmark result using `msr("regr.mse")`.
 
 ```{r solutions-041}
 library(mlr3benchmark)
 bma = as_benchmark_aggr(bmr, measures = msr("regr.mse"))
 bma
 ```
 
-We can now use the `$friedman_test()` method to apply the test to the experiment results.
-We do not need a post-hoc test, because we are only comparing two algorithms.
+We can also visualize this result using the `r ref("mlr3benchmark::autoplot()")` function.
 
+```{r}
+autoplot(bma)
+```
+
+Below, we conduct a global Friedman test.
+Note that a post-hoc test is not needed because we are only comparing two algorithms.
```

Comment:
...but on multiple datasets?

Reply:
But the available post-hoc test only does a comparison of all pairs of algorithms across all datasets.